sensors/src/main/scala/com/lynbrookrobotics/potassium/sensors/imu/IMUValue.scala

```diff
@@ -0,0 +1,6 @@
+package com.lynbrookrobotics.potassium.sensors.imu
+
+/**
+  * Constructs a single datapoint from the IMU.
+  */
+case class IMUValue(gyro: Value3D[Double], accel: Value3D, magneto: Value3D) {}
```

Comment:
This wouldn't compile. `gyro` should be a `Value3D[Angle]`, `accel` should be a `Value3D[Acceleration]`, and `magneto` should be a `Value3D[MagneticFluxDensity]`.